digital-ocean/digital-ocean.lua

```diff
@@ -0,0 +1,172 @@
+--[[
+
+# [Digital Ocean API](https://developers.digitalocean.com/documentation/v2/) connector.
+
+### Authorization
+
+Authorization use [API Key](https://cloud.digitalocean.com/settings/applications).
```

Comment:
Here is the proper link for https://developers.digitalocean.com/documentation/v2/#authentication
And its not "API Key" type, but it stated in the documentation "The DigitalOcean API handles this through OAuth". So should be OAuth in the anchor.